definitions/invoice/tax.yaml

```diff
@@ -0,0 +1,94 @@
+# Tax
+type: object
+description: Tax Information
+required:
+  - tx_id
+  - tx_id_number
+  - tax_type
+  - tax_tiers
+  - taxing_jurisdiction_country
+  - tax_on_tax
+  - is_increase
+properties:
+  tx_id:
+    type: string
+    description: |
+      A unique identifier associated with the tax/rate/jurisdiction, assigned by the law firm.
+    example: T1234556
+  tx_id_number:
```

Comment:
I think its tax_id_number not tx_id_number?

Reply:
Cheers. I think in the XML spec they have a mix between the two (cli_tax_id, tax_on_tax_id but tx_id) - think it's one of things they want to change but for now I'll stick with the spec.